src_c/event.c

```diff
@@ -1617,9 +1703,9 @@ static PyMethodDef _event_methods[] = {
     {"pump", (PyCFunction)pg_event_pump, METH_NOARGS, DOC_PYGAMEEVENTPUMP},
     {"wait", (PyCFunction)pg_event_wait, METH_NOARGS, DOC_PYGAMEEVENTWAIT},
     {"poll", (PyCFunction)pg_event_poll, METH_NOARGS, DOC_PYGAMEEVENTPOLL},
-    {"clear", pg_event_clear, METH_VARARGS, DOC_PYGAMEEVENTCLEAR},
-    {"get", pg_event_get, METH_VARARGS, DOC_PYGAMEEVENTGET},
-    {"peek", pg_event_peek, METH_VARARGS, DOC_PYGAMEEVENTPEEK},
+    {"clear", pg_event_clear, METH_VARARGS | METH_KEYWORDS, DOC_PYGAMEEVENTCLEAR},
```

Comment:
You may need to cast these methods as `(PyCFunction)` to avoid compiler warnings.

Ref: https://docs.python.org/3/c-api/structures.html#c.PyMethodDef